### PR TITLE
[plugin.video.zoneminder] 3.0.0

### PR DESCRIPTION
**Pull request by petegallagher**
### Description
Update Zoneminder plugin for Kodi 19/Python 3.

### Checklist:
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.zoneminder/addon.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <import addon="script.module.requests" version="2.7.0"/>
+</requires>
+  <extension point="xbmc.python.pluginsource" library="zoneminder.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <platform>all</platform>
+    <summary lang="en_GB">View ZoneMinder video surveillance system</summary>
+    <description lang="en_GB">View the live stream of cameras via the Zoneminder video surveillance system.</description>
+    <license>GPL-3.0</license>
```
Comment (enen92) [inline plugin.video.zoneminder/addon.xml]: GPL-3.0-only or GPL-3.0-or-later (please use a valid spdx identifier)

File: plugin.video.zoneminder/resources/settings.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<settings>
```
Comment (enen92) [inline plugin.video.zoneminder/resources/settings.xml]: It'd be nice if you could update the settings to the new format. You could use https://gitlab.com/ronie/script.settings-convert for an automatic conversion

**Review by enen92 — CHANGES_REQUESTED** [review]
Just the minor on the license

**Comment by petegallagher** [comment]
@enen92 assuming this PR meets your requirements, can you squash these additional commits on merge?

**Review by enen92 — APPROVED** [review]

**Pull request merged**